data_structures/src/superblock.rs

```diff
@@ -552,23 +561,20 @@ mod tests {
         let bls_pk =
             Bn256PublicKey::from_secret_key(&Bn256SecretKey::from_slice(&[1; 32]).unwrap())
                 .unwrap();
-        let sb1 = sbs
-            .build_superblock(
-                &block_headers,
-                &ars_identities,
-                &ars_identities,
-                &[bls_pk],
-                100,
-                0,
-                genesis_hash,
-            )
-            .unwrap();
+        let sb1 = sbs.build_superblock(
+            &block_headers,
+            &ars_identities,
+            &[bls_pk],
+            100,
+            0,
+            genesis_hash,
+        );
         let v1 = SuperBlockVote::new_unsigned(sb1.hash(), 0);
         assert_eq!(sbs.add_vote(&v1), AddSuperBlockVote::NotInSigningCommittee);
     }
 
     #[test]
-    fn superblock_state_first_superblock_none() {
+    fn superblock_state_first_superblock_empty() {
         // If the first superblock is None, the state is not updated except for the superblock_index
         let mut sbs = SuperBlockState::default();
 
```

Comment:
i think the comment below should be updated to "If there were no blocks, there will be an empty superblock"

Reply:
True... it got lost during the rebase... :rofl: